src/main/java/com/gistpetition/api/petition/presentation/PetitionController.java

```diff
@@ -36,14 +36,30 @@
     }
 
     @GetMapping("/petitions")
-    public ResponseEntity<Page<PetitionPreviewResponse>> retrievePetitions(@RequestParam(defaultValue = "0") Long categoryId,
-                                                                           @PageableDefault(sort = "createdAt", direction = Sort.Direction.DESC) Pageable pageable) {
+    public ResponseEntity<Page<PetitionPreviewResponse>> retrieveReleasedPetitions(@RequestParam(defaultValue = "0") Long categoryId,
+                                                                                   @PageableDefault(sort = "createdAt", direction = Sort.Direction.DESC) Pageable pageable) {
+        if (categoryId.equals(0L)) {
+            return ResponseEntity.ok().body(petitionService.retrieveReleasedPetition(pageable));
+        }
+        return ResponseEntity.ok().body(petitionService.retrieveReleasedPetitionByCategoryId(categoryId, pageable));
+    }
+
+
+    @GetMapping("/petitions/all")
```

Comment:
```suggestion
    @ManagerPermissionRequired
    @GetMapping("/petitions/all")
```
매니저 이상만 접근 가능하도록 해야할 것 같아요!

Reply:
우선은 개발의 편의를 위해서 열어둔 상태긴 합니다! 이 api에 대한 부분은 프론트 측과 이야기 해보고 결정을 진행했으면 해요!

좋은 의견 감사합니다!